infra/docker/train_multinode.sh

```diff
@@ -43,11 +50,17 @@ DATA.TRAIN='["train2017"]' \
 DATA.VAL='("val2017",)' \
 TRAIN.BATCH_SIZE_PER_GPU=${BATCH_SIZE_PER_GPU} \
 TRAIN.LR_EPOCH_SCHEDULE='[(8, 0.1), (10, 0.01), (12, None)]' \
-TRAIN.EVAL_PERIOD=12 \
+TRAIN.EVAL_PERIOD=24 \
 RPN.TOPK_PER_IMAGE=True \
-PREPROC.PREDEFINED_PADDING=True \
+PREPROC.PREDEFINED_PADDING=False \
 BACKBONE.WEIGHTS=/data/pretrained-models/ImageNet-R50-AlignPadding.npz \
 BACKBONE.NORM=FreezeBN \
+TRAIN.BACKBONE_NCHW=False \
+TRAIN.FPN_NCHW=False \
+TRAIN.RPN_NCHW=False \
+TRAIN.MASK_NCHW=False \
+TRAIN.WARMUP_INIT_LR=0.000416666666667 \
+FRCNN.BBOX_REG_WEIGHTS='[20., 20., 10., 10.]' \
+TRAIN.GRADIENT_CLIP=0 \
 TRAINER=horovod
-#For 32x4
-#TRAIN.GRADIENT_CLIP=1.5
+# set to 1.5 to enable clip
```

Comment:
Maybe extend comment here saying loss can be NaN if gradient clip is not set for large batch sizes

Reply:
Done